cumulus_library/template_sql/extension_denormalize.sql.jinja

```diff
@@ -72,7 +72,7 @@ CREATE TABLE {{ target_table }} AS (
             {%- endif %}
             ROW_NUMBER()
                 OVER (
-                    PARTITION BY id, system
+                    PARTITION BY id
```

Comment:
This is a bug fix, afaict - when you have both ombCategory and detailed, this was partitioning in such a way that both would have available_priority = 1 and both show up in the extension table. Then when you joined with patients, you'd get duplicate patient rows.

Reply:
We talked about this on slack - we think it _is_ a bug, and will leave the code change in. But we have lots of questions about best approaches for race/ethnicity. Should `race_detailed` be a separate column? What to do with `text`? Problems for another day.